src/App.css

```diff
@@ -9,13 +9,14 @@
 
 .App-header {
   background-color: #222;
-  height: 50px;
-  padding: 20px;
+  /* height: 50px; */
+  /* padding: 20px; */
```

Comment:
Remember to remove commented code after you are happy with a change, you can always come back to see the previous state of a file using git :)